lib/last_login_sync/gateway/activity.rb

```diff
@@ -0,0 +1,9 @@
+class LastLoginSync::Gateway::Activity
+  def since(date:)
+    sessions = Session.select(:username)
+      .where { start < date.next_day }
+      .where { start >= date }
+      .distinct(:username)
+    sessions.map(&:username)
```

Comment:
Wont .select return a list of usernames anyway?

Reply:
Could you show me what this query evaluates to in SQL?

Reply:
It returns something akin to a Hashmap, so we need a `map` here to make it a list